client/components/card/compact.jsx

```diff
@@ -3,14 +3,14 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	assign = require( 'lodash/assign' ),
-	classnames = require( 'classnames' );
+import React from 'react';
+import assign from 'lodash/assign';
```

Comment:
Noted elsewhere too, but I'd recommend changing this to `import { assign } from 'lodash';` while we're here.